common/test/acceptance/pages/studio/asset_index.py

```diff
@@ -88,3 +93,371 @@ def return_results_set(self):
         Returns the asset set from the page
         """
         return self.q(css="#asset-table-body tr").results
+
+
+class AssetIndexPageStudioFrontend(CoursePage):
+    """The Files and Uploads page for a course in Studio"""
+
+    PAGINATION_PAGE_ELEMENT = ".pagination li"
+    TABLE_SORT_BUTTONS = 'th.sortable button.btn-header'
+    TYPE_FILTER_ELEMENT = ".filter-set .form-group"
+    URL_PATH = "assets"
+
+    @property
+    def url(self):
+        """Construct a URL to the page within the course."""
+        # TODO - is there a better way to make this agnostic to the underlying default module store?
+        default_store = os.environ.get('DEFAULT_STORE', 'draft')
+        course_key = CourseLocator(
+            self.course_info['course_org'],
+            self.course_info['course_num'],
+            self.course_info['course_run'],
+            deprecated=(default_store == 'draft')
+        )
+        url = "/".join([BASE_URL, self.URL_PATH, urllib.quote_plus(unicode(course_key))])
+        return url if url[-1] == '/' else url + '/'
+
+    @wait_for_js
+    def is_browser_on_page(self):
+        return all([
+            self.q(css='body.view-uploads').present,
+            self.q(css='.page-header').present,
+            self.q(css='#root').present,
+            not self.q(css='div.ui-loading').visible,
+        ])
+
+    @wait_for_js
+    def is_sfe_container_on_page(self):
```

Comment:
I don't think this is being called anywhere. Is it left over? 🤷‍♂️

Reply:
I think we should call it. I guess I might have missed it in a refactor. I'll add it back in.